app/models/case_request.rb

```diff
@@ -5,10 +5,10 @@ class CaseRequest
     :confirmation_code,
     :fees
 
-  def initialize(opts = {})
-    @case_reference = opts[:case_reference]
-    @confirmation_code = opts[:confirmation_code]
-    @fees = []
+  def initialize(case_reference, confirmation_code)
+    @case_reference    = case_reference
+    @confirmation_code = confirmation_code
+    @fees              = []
```

Comment:
Personally, I'd prefer to stay away from this style of alignment.  I've found it more painful in refactoring and merging.